### ISSUE TITLE
False positives if trailing whitespace in multiline list is found

### ISSUE THREAD

**Issue by priv-kweihmann** [priv-kweihmann/oelint-adv#179]
```
DEPENDS += "\
 a \
 b \ 
 c \
"
```
produces 
* Line shall not begin with a space
* Variable value should be quoted
* No spaces after line continuation
* Line shall not end with a space

while only the 3rd one is really meaningful
